zenodo/modules/metrics/config.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2017-2021 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Configuration for Zenodo Frontpage."""
+
+from __future__ import absolute_import, print_function
+from . import api
+
+api = api.ZenodoMetric()
+
+ZENODO_METRICS = {
+    'openaire-nexus': [
+        {
+            'name': 'kpi.srvs.UsersZenodo',
+            'help': 'Number of unique users of Zenodo',
+            'type': 'gauge',
+            'value': api.get_users(cache_timeout=3600)
+        },
```

Comment:
```suggestion
ZENODO_METRICS = {
    'openaire-nexus': [
        {
            'name': 'kpi.srvs.UsersZenodo',
            'help': 'Number of unique users of Zenodo',
            'type': 'gauge',
            'cache_timeout': 3600,
            'value': ZenodoMetric.get_users,
        },
```

- Since it's a class `staticmethod` you can specify it like this (i.e. no need for an instance)
- The `cache_timeout` can be moved to be a config key